tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: prepare-repo
+      image: quay.io/konflux-ci/release-service-utils:4576e1e2a30439129cb69c8a4f39f7ced2d44376
+      script: |
+        set -eux
```

Comment:
Can you add a shebang? It's not strictly required, but including it makes it more explicit and also consistent with most other tasks. I would recommend `#!/usr/bin/env bash`. This applies to other steps as well. I can see the mocks file already has it.

Reply:
Will fix.

Reply:
Fixed in new commit.

Reply:
Added a new comment here: https://github.com/konflux-ci/release-service-catalog/pull/510/files#r1753909010